src/v/compat/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+cmake_minimum_required(VERSION 3.12.0)
+
+if (NOT EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/bin)
+  execute_process(
+    COMMAND "/usr/bin/git" clone -b main https://github.com/felixguendling/rp-compat-bin.git bin
+    WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
+  )
+endif ()
+
+if (NOT EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/corpus)
+  execute_process(
+    COMMAND "/usr/bin/git" clone -b main https://github.com/felixguendling/rp-compat-corpus.git corpus
```

Comment:
@felixguendling should we fork this for redpanda specifically, or this is a simple util.

Reply:
This repository should become part of the redpanda-data organization on GitHub. Its purpose is to contain binary data from all release versions of redpanda together with JSON files that contain the same data. This way it's possible to check if all versions of repdanda that should be compatible can actually read the data back. The deserialized values have to match the contents stored in the JSON file for this test to be successful.

This repository has the purpose to store all the binary and JSON files produces by all versions of redpanda.

Reply:
Makes total sense.

Reply:
Transfer it to me and I’ll transfer it to the organization